src/main/cpp/apple_fsnotifier.cpp

```diff
@@ -7,6 +7,20 @@
 
 using namespace std;
 
+struct FileWatcherException : public exception {
+public:
+    FileWatcherException(const char *message) {
+        this->message = message;
+    }
+
+    const char * what () const throw () {
```

Comment:
```suggestion
    const char* what() const throw() {
```

Nit. What code style shall we use here? Can we maybe configure VSCode to autoformat the native code?